README.md

```diff
@@ -1,14 +1,14 @@
 # MotoReMinder
 
-MotoReMinder is a universal application that will remind automobile owners when various maintenance intervals are passed based on mileage driven. If the user has access to a Bluetooth enabled OBD-II scanner and their car supports it, the mileage on the car will be automatically updated in the background and push-notifications will be sent to their device to remind them of the intervals coming up.
+### MotoReMinder is a universal application that will remind automobile owners when various maintenance intervals are passed based on mileage driven. If the user has access to a Bluetooth enabled OBD-II scanner and their car supports it, the mileage on the car will be automatically updated in the background and push-notifications will be sent to their device to remind them of the intervals coming up.
```

Comment:
This use of section header format for an entire paragraph seems inappropriate.

Who is your intended reader? Are they necessarily familiar with the phrase "universal application"? This is one case where I think getting a GPT rewrite can be useful by presenting alternative ways to communicate the information to average drivers.

Reply:
I appreciate the feedback! I'll definitely check out some LLM options and make another pull with an improved section